en/deploy/README.md

```diff
@@ -40,14 +40,14 @@ If it isn't installed already, git should be available via your package manager,
 
 Git tracks changes to a particular set of files in what's called a code repository (or "repo" for short). Let's start one for our project. Open up your console and run these commands, in the `djangogirls` directory:
 
-> **Note**: Check your current working directory with a `pwd` (OSX/Linux) or `cd` (Windows) command before initializing the repository. You should be in the `djangogirls` folder.
+> **Note** Check your current working directory with a `pwd` (OSX/Linux) or `cd` (Windows) command before initializing the repository. You should be in the `djangogirls` folder.
 
     $ git init
     Initialized empty Git repository in ~/djangogirls/.git/
-    $ git config user.name "Your Name"
-    $ git config user.email you@example.com
+    $ git config --global user.name "Your Name"
```

Comment:
@hjwp, any comments?

As I sad on the chat:
> Every guide on the internet has --global; I think it's a very sensible thing to tell beginners to do.
> I don't use --default for user.email because I use different addresses for work and non-work projects, but that's an advanced use case.